Update Brower Log collection instructions (#4479)
DataDog/documentation: content/en/logs/log_collection/javascript.md

      // OPTIONAL
      // add global metadata attributes

>       Datadog.addLoggerGlobalContext({<META_KEY>: '<META_VALUE>'});

Reviewer: ```suggestion
      Datadog.addLoggerGlobalContext({<META_KEY>: <META_VALUE>});
```
`<META_VALUE>` does not have to be a string

}  
```

## Advanced usage

### Filter by severity

> In some cases, you might want to disable the debug mode or to only collect warning and errors. This can be achieved by changing the logging level thanks to the `logLevel` parameter to `debug`, `info`, `warn`, and `error` :

Reviewer 2: ```suggestion
In some cases, you might want to disable the debug mode or only to collect warning and errors. This can be achieved by changing the logging level thanks to the `logLevel` parameter to `debug`, `info`, `warn`, and `error` :
```

this is called a split infinitive

```
Datadog.logger.setLogLevel('<SEVERITY_LEVEL>')
```

> Only logs with a severity equal or higher to the specified one are sent.

Reviewer: could it be useful to specify that the default log level is `debug`?

### Change the destination

> By default, the loggers are sending logs to Datadog. It is also possible to configure the logger to send logs to the console or to not send logs at all. This can be used in development environment to keep the logs locally.

Reviewer 2: ```suggestion
By default, the loggers are sending logs to Datadog. It is also possible to configure the logger to send logs to the console or not to send logs at all. This can be used in the development environment to keep the logs locally.
```

Use the `setLogHandler` function with the values `http`, `console`, or `silent`:
```
Datadog.logger.setLogHandler('<HANDLER>')
```

### Define multiple loggers

> The library contains a default logger but it is also possible define different loggers which can be convenient when several team are working on the same project.

Reviewer 2: ```suggestion
The library contains a default logger, but it is also possible to define different loggers which can be convenient when several teams are working on the same project.
```

> Each logger can optionally be configure with its own log level, handler and context. Note that the `Global Context` is added on top of each logger context. 

Reviewer 2: ```suggestion
Each logger can optionally be configured with its own log level, handler, and context. Note that the `Global Context` is added on top of each logger context. 
```

Use the following to define a custom logger:

```
createLogger (<LOGGER_NAME>, {
    logLevel?: 'debug' | 'info' | 'warn' | 'error'
    logHandler?: 'http' | 'console' | 'silent'
    context?: { <KEY>:'<VALUE>'}
})
```

Those parameters can also be set thanks to the `addContext`, `setLogLevel`, and `setLogHandler` functions.

> You can then get this logger in any part of your Javascript code thanks to the `getLogger` function:

Reviewer: ```suggestion
After the creation of this logger, you can then get it in any part of your Javascript code thanks to the `getLogger` function:
```

```
const my_logger = getLogger('<LOGGER_NAME>') 
```

**Example:**

Let's assume that there is a signup logger define with all the others logger:

```
# create a new logger
const signupLogger = createLogger('signupLogger'})
signupLogger.addContext({ env: 'staging'})
```

> It can now be used in different part of the code with:

Reviewer 2: ```suggestion
It can now be used in a different part of the code with:
```
